python/ray/data/tests/test_dataset.py

```diff
@@ -1969,12 +1969,12 @@ def test_iter_batches_basic(ray_start_regular_shared):
 
     # Batch size larger than dataset.
     batch_size = 15
-    batches = list(
-        ds.iter_batches(batch_size=batch_size, batch_format="pandas"))
+    batches = list(ds.iter_batches(batch_size=batch_size, batch_format="pandas"))
     assert all(len(batch) == ds.count() for batch in batches)
     assert len(batches) == 1
-    assert pd.concat(
-        batches, ignore_index=True).equals(pd.concat(dfs, ignore_index=True))
+    assert pd.concat(batches, ignore_index=True).equals(
+        pd.concat(dfs, ignore_index=True)
+    )
```

Comment:
accidental changes?

Reply:
Not quite– for some reason the linter was breaking in the last run because this change didn't exist. Now the linter is passing, so I believe this is necessary.

Reply:
Ah it was fixed in master